### PR TITLE
added some sanity checking to work with PMA/mysqldump exports that don't...

### PR DESCRIPTION
**Pull request by axfelix**
... contain a CREATE DATABASE statement

### CODE REVIEW

File: start-database.sh [hunk 1]
```diff
@@ -8,7 +8,11 @@ echo "Adding data into MySQL"
 file -bi /var/mysql/database.sql
 /usr/sbin/mysqld &
 sleep 5
-curl $url | mysql --default-character-set=utf8
+curl $url -o import.sql
+sed -ri.bak 's/-- Database: (.*?)/CREATE DATABASE \1/g' import.sql
```
Comment (nkratzke) [inline start-database.sh]: I think the CREATE DATABASE statement has to close with a ; to generate valid SQL code. SQL statements have to be separated by a ';'

File: start-database.sh [hunk 2]
```diff
@@ -8,7 +8,11 @@ echo "Adding data into MySQL"
...
+if grep -q "CREATE DATABASE" file.sql; then :; else sed -ri.bak 's/-- Table structure/CREATE DATABASE `database_1`\n-- Table Structure/g' import.sql; fi
```
Comment (nkratzke) [inline start-database.sh]: I do not understand the purpose of this line here. Why you are insert before every table definition a new CREATE DATABASE statement? There are a plenty of CREATE DATABASE statements in import.sql. CREATE DATABASE should be handled with line 12 already?

Why yo are grepping 'file.sql'. Where does this file come from? Do you mean 'import.sql'?

**Comment by nkratzke** [comment]
Would it be possible to fix the above mentioned issues? I checked your contribution by your provided test file. And it produced no processable/valid SQL (the Database 'database_1' is created several times for example).

**Comment by axfelix** [comment]
sorry about that -- I butchered it trying to rebase (as usual) and reforked. was going to open a new PR instead.

**Pull request merged**